core-service/src/main/java/org/fundacionjala/sevenwonders/beans/GameRoomService.java

```diff
@@ -61,18 +66,16 @@ public GameModel getGameRoom(int id){
      *
      * @return GameRooms
      */
-    public Collection<GameModel> listGameRooms(){
-
-        List<GameModel> currentGameRoomModels = new ArrayList<>();
-        for (int i = 1; i <= gameRooms.size(); i++) {
-            GameModel room = new GameModel();
-            room.setId(i);
-            room.setName(gameRooms.get(i).getName());
-            room.setPlayer(gameRooms.get(i).getPlayers().get(0));
-            room.setMaxPlayers(gameRooms.get(i).getMaxPlayers());
-            room.setPlayers(gameRooms.get(i).getPlayers());
+    public Collection<GameRoomModel> listGameRooms(){
+        List<GameRoomModel> currentGameRoomModels = new ArrayList<>();
+        gameRooms.entrySet().stream().forEach(entry -> {
+            GameRoomModel room = new GameRoomModel();
```

Comment:
The returned room doesn't contain the gameroom name which is used by the lobby ui